src/main/java/com/Novi/TechItEasy/models/Television.java

```diff
@@ -51,39 +49,25 @@ public class Television {
     @Column(name = "sold")
     private int sold;
 
+    @JsonIgnore
+    @OneToOne
+    private RemoteController remoteController;
 
-    public Television() {
-    }
+    @JsonIgnore
+    @ManyToOne
+    private CI_Module ci_module;
+
+    @JsonIgnore
+    @ManyToMany
```

Comment:
Dit is prima, maar hoe ga je dit nu vullen in je data.sql? Wat is de naam van de tussentabel waarin je gaat inserten? Het is makkelijker om met @JoinTable wel de naam van de tabel te zetten en eventueel ook de namen van de kolommen